src/components/helpers/assoc-list-item-value.js

```diff
@@ -32,7 +32,11 @@ export default createReactClass({
     const field = this.props.field;
 
     const fieldElem = config.createFieldElement({
-      field,
+      field: {
+        ...field,
+        id: `${this.props.id}_${this.props.index}`,
+      },
+      labelledby: this.props.id ? `${this.props.id}_label` : undefined,
```

Comment:
Same question as the key above. Is it appropriate to have multiple fields labeled by the same label?